metrics/storage/fio_job.sh

```diff
@@ -91,6 +92,10 @@ function create_fio_prep_job()
 		--filename="$FILE_NAME" \
 		--bs="$BLOCK_SIZE" \
 		--size="$FILE_SIZE" \
+		--time_based \
+		--direct=1 \
+		--invalidate=1 \
+		--ramp_time="$RAMP_TIME" \
```

Comment:
Can you add a new command line option so we can optionally change RAMP_TIME please.

Reply:
@grahamwhaley done